------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-- CloneTrooper1019, 2020
-- Realism Config
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

local REALISM_CONFIG = {
	---------------------------------------------
	-- Multiplier values (in radians) for each
	-- joint, based on the pitch/yaw look angles
	---------------------------------------------

	RotationFactors = {

		Head = {
			Pitch = 0,
			Yaw = 0,
		},
	},
}

return REALISM_CONFIG
